From fcbeaf85fe7e73cc850f6b538c669753c3fb0f89 Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz@gentoo.org>
Date: Fri, 18 Sep 2026 00:46:17 -0400
Subject: [PATCH] remove terrible horrible no-good very bad profile

Overriding the profile means make.conf is not respected, and also blows
up.

---
 Cargo.toml | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 37121ae..d8a2f74 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -88,11 +88,6 @@ name = "gpg-recorder"
 path = "src/gpg-recorder.rs"
 required-features = ["the-recorder-steals-secrets-and-i-wont-package-it-in-slash-bin"]
 
-[profile.release]
-codegen-units = 1
-debug = true
-lto = true
-
 [features]
 default = ["crypto-nettle"]
 crypto-nettle = ["sequoia-openpgp/crypto-nettle"]
-- 
2.55.0

